Lesson16/selcan/script.js

```diff
@@ -0,0 +1,304 @@
+
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+function isGreaterNumber(sayi1,sayi2){
+	if (sayi1 > sayi2){
+		console.log("Exercise3;",sayi1 , sayi2 ,"'den büyükktür.");
+	} else if(sayi1 === sayi2){
+		console.log("Exercise3;",sayi1 , sayi2 ,"'ye eşittir.") ;
+	} else{
+		console.log("Exercise3;",sayi1 , sayi2 ,"'den küçüktür.") ;
+	}
+
+}
+isGreaterNumber(11,45);
+
+// 4. Write a function that takes in a string and checks if the first letter is uppercase using if else statement.
+function isFirstLetterUppercase(stringToCheck){
+const firstLetter = stringToCheck[0];
+  if (firstLetter === firstLetter.toUpperCase()) {
+    return true;
+  } else {
+    return false;
+  }
+
+}
+console.log(isFirstLetterUppercase("Televiision"));
+
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function ifPositive(number){
+	if(number > 0){
+		return "Positive";
+	}else if(number===0){
+		return "Zero";
+	}else{
+		return "negative";
+	}
+}
+console.log(Exercise5,ifPositive(76));
+
+// 6. Write a function that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function isYears(year){
+	if (year % 4 === 0 && year % 100 !== 0 && year % 400 !== 0)
+	{
+        return "It is a leap year."
+	} else {
+		return "It isn't a leap year."
+	}	
+}
+console.log(isYears(2018));
+
+// 7. Write a function that takes in a temperature and checks if it's above or below freezing using if else statement.
+function isTemperatureAbove(degree){
+	if (degree < 0){
+		return "Exercise7;It is below freezing";
+	} else {
+		return "Exercise7; It is above freezing"
+	}
+}
+console.log(isTemperatureAbove(4));
+
+// 8. Write a function that takes in a person's height and weight and checks if they are considered underweight, normal, overweight, or obese using if else statement.
+// Body Mass Index (BMI) is a person’s weight in kilograms (or pounds) divided by the square of height in meters (or feet). A high BMI can indicate high body fatness.
+//Formula: weight (kg) / [height (m)]2
+// Below 18.5	Underweight
+// 18.5 – 24.9	Healthy Weight
+// 25.0 – 29.9	Overweight
+// 30.0 and Above	Obesity
+
+function BMI(weight,height){
+let sonuc = weight / height*height;
+if (sonuc <18.5){
+	return "Exercise8: Underweight";
+} else if (sonuc >=18.5 && sonuc <24.9){
+	return "Exercise8: Healthy Weight";
+} else if (sonuc >=25 && sonuc <29.9){
+	return "Exercise8: Overweight";
+} else if (sonuc >=30){
+	return "Exercise8: Obesity";}
+}
+console.log(BMI(90,1.70));
+
+// 9. Write a function that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
+function isChild(age){
+	if (age>0 && age<12){
```

Comment:
In this solution, you need to use >= and <= to make sure that the edge values are included (e.g. the person with age 12 is a child)
```suggestion
	if (age>=0 && age=<12){
```

Reply:
Same for the other case, all of the should use either >= or <=